app-dicts/myspell-ru/myspell-ru-20180627.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Could you try with EAPI=7? The eclass should support it, but I havent tested so cant be 100 % sure.

Reply:
> Could you try with EAPI=7? The eclass should support it, but I havent tested so cant be 100 % sure.

EAPI=7 is okay